config/routes.rb

```diff
@@ -102,6 +102,7 @@
           resources :listings, only: [:index], to: "organizations#listings"
           resources :articles, only: [:index], to: "organizations#articles"
         end
+        resources :forem_directories, only: %i[index]
```

Comment:
I agree with @rhymes above. This seems like it makes sense as a singular resource.